clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy

```diff
@@ -200,6 +200,7 @@ class OpenstackServerGroupCachingAgent extends AbstractOpenstackCachingAgent imp
   OpenstackServerGroup buildServerGroup(ProviderCache providerCache, Stack stack, Set<String> loadbalancerIds) {
     Map<String, Object> launchConfig = buildLaunchConfig(stack.parameters)
     Map<String, Object> openstackImage = buildImage(providerCache, (String) launchConfig?.image)
+    Map<String, Object> advancedConfig = buildAdvancedConfig(stack.parameters)
 
     //TODO this check will change once we have a config that indicates the openstack version, e.g. kilo, liberty, mitaka
     //kilo stack create times have a 'Z' on the end. It was removed in liberty and mitaka.
```

Comment:
Oh! We don't support Kilo anymore, can you remove the Z with the timestamp? In a different PR of course :)